sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,5 @@
+select "cunt"."name",
+        count("cit"."cityId") as "numOfCities"
+  from "cities" as "cit"
+  join "countries" as "cunt" using ("countryId")
+  group by "cunt"."name"
```

Comment:
No matter how British you're feeling, it's probably a good idea to refrain from cursing in your source code.